### ISSUE TITLE
Rename to Kiwix

### ISSUE THREAD

**Issue by rgaudin** [kiwix/kiwix-desktop#1243]
At this stage, that could be classified as a bug I think 😀

According to the [branding document](https://github.com/kiwix/overview/wiki/Branding), `kiwix-desktop` should be exposed as `Kiwix`.
End user should never see `kiwix-desktop` except in a link to the repository.

That means renaming the App so the Title bar says `Kiwix`, renaming the binaries (`Kiwix[.exe]`) and I believe the Archive names but I'm not clear on what it should actually be: `Kiwix Windows_x64_2.4.0.zip` ?? (see https://github.com/kiwix/overview/issues/106 as well)

Side question for @kelson42 and @Popolechien. Are you OK with having both `Kiwix Windows` (ie `Kiwix` + Platform) and `Kiwix Hotspot` (ie. a Kiwix product) ?

**Comment by Popolechien** [comment]
Yes to the first question (Kiwix-desktop -> Kiwix), but I am not sure I understand the other tbh. Shouldn't it be `Kiwix for Windows` ?

**Comment by rgaudin** [comment]
> Yes to the first question (Kiwix-desktop -> Kiwix)

That was not a question 😉

> but I am not sure I understand the other tbh. Shouldn't it be `Kiwix for Windows` ?

That's the question. The branding doc says that all readers are `Kiwix` from their own perspective, and there is no debate AFAIK.
To the outside world though, we need to distinguish the different products and the doc says we should have

- Kiwix Windows
- Kiwix Linux
- Kiwix Android
- Kiwix Hotspot

This is only marginally relevant to this ticket as it would only apply to the _packaging_ (the archive) to download kiwix-desktop and that happens on kiwix-build but as I wrote, we might want to consider updating the branding doc as I find it odd to have both `Kiwix {Platform}` and `Kiwix Hotspot` using the same pattern. Your suggestion makes sense (`Kiwix for {Platform}`).

**Comment by kelson42** [comment]
@rgaudin yes, yes yes ;)

**Comment by Popolechien** [comment]
Looks like we're all in agreement then:

- `Kiwix-desktop` displayed as `Kiwix`
- Tweak product list with `for` (e.g. `Kiwix for Windows`, except the `Kiwix Hotspot`, which is a different product approach)
- Repositories remain as-is.

**Comment by rgaudin** [comment]
OK then.
